src/test/java/io/supertokens/test/passwordless/PasswordlessDevicesTest.java

```diff
@@ -105,22 +125,29 @@ public void getDevicesFromHashIDWithCodes() throws Exception {
      * @throws Exception
      */
     @Test
-    public void getDevicesFromInvalidHashID() throws Exception {
+    public void getDevicesFromInvalidID() throws Exception {
```

Comment:
I'd prefer if the test names matched the tested func.

Reply:
done.